lib/elixir_google_scraper/accounts.ex

```diff
@@ -0,0 +1,352 @@
+defmodule ElixirGoogleScraper.Accounts do
+  @moduledoc """
+  The Accounts context.
+  """
+
+  import Ecto.Query, warn: false
+
+  # Disable it due to https://github.com/rrrene/credo/issues/441
+  # credo:disable-for-next-line Credo.Check.Consistency.MultiAliasImportRequireUse
+  alias ElixirGoogleScraper.Accounts.{User, UserNotifier, UserToken}
```

Comment:
Later if we're using this often, we can disable [MultiAliasImportRequireUse](https://github.com/rrrene/credo/issues/320) completely 👍 .